test/end-to-end/offer.js

```diff
@@ -286,6 +291,35 @@ describe("Offer endpoint tests", () => {
                 await Offer.deleteMany({});
             });
 
+            beforeEach(async () => {
+                await Offer.deleteMany({});
+            });
+
+            test("Should fail to create an offer due to publish end date being aftr publish date more than the limit", async () => {
+                const offer = generateTestOffer();
+                const publishDate = new Date(Date.now() - (DAY_TO_MS));
+                const offer_params = {
+                    ...offer,
+                    owner: test_company._id,
+                    ownerName: test_company.name,
+                    publishDate: publishDate.toISOString(),
+                    publishEndDate: (new Date(publishDate.getTime() + (MONTH_IN_MS * OFFER_MAX_LIFETIME_MONTHS) + DAY_TO_MS)).toISOString(),
+                };
+
+                const res = await request()
+                    .post("/offers/new")
+                    .send(withGodToken(offer_params));
+
```

Comment:
I meant to only remove the console.info, I don't know why the comment had multiple lines.

Add the status check as it were
`expect(res.status).toBe(HTTPStatus.UNPROCESSABLE_ENTITY);`